app/src/debug/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,16 @@
+<network-security-config>
```

Comment:
just add this for debugging purpose, for example we can mock the response using tool like https://proxyman.com/ or https://www.charlesproxy.com/